src/App.js

```diff
@@ -1,12 +1,26 @@
 import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route
+} from "react-router-dom";
+
+import './App.scss';
```

Comment:
split styles and component to different blocks
// styles
....

// components
....